web/src/main/java/org/fao/geonet/kernel/search/classifier/ClassificationScheme.java

```diff
@@ -0,0 +1,129 @@
+//=============================================================================
```

Comment:
Not sure what you are referring to here web/src/main/java/org/fao/geonet/kernel/search/classifier/ClassificationScheme.java or the Geonetwork copyright notice.